README.md

```diff
@@ -21,10 +21,16 @@ A shiny app that provides visual representation of fish tracks within the beam a
+In the future these files may be included in the repo using Git Large File Storage (LFS) protocol.  Details can be found at [Git LFS](https://git-lfs.github.com/). LFS Files can still be read directly from the repo by using the link obtained by viewing *raw* data. A full tutorial on how to use Git LFS is located [HERE](https://github.com/git-lfs/git-lfs/wiki/Tutorial#migrating-existing-repository-data-to-lfs). Prior to cloning the git repository to your local drive, it's necessary to have Git-LFS downloaded on your physical computer. If your local directory is not showing the large files you're looking for, ensuring the LFS software is installed should be your first troubleshooting step. Follow download steps at [Git LFS](https://git-lfs.github.com/).
 
 ## Creating the model data
 1. Data has been processed in Echoview (v. 13) and exported as csv files. The raw acoustic exports are located in [Data](Data) with each fish having it's own folder.  
-2. The exported acoustic data is compiled to a master data set using [Analysis_Scripts/read_tidy_export_EVfiles.R](Analysis_Scripts/read_tidy_export_EVfiles.R). This script will import and format each fish's exported data and then compile a large dataframe [ProcessedData/processed_AllFishCombined_unfiltered.csv](ProcessedData/processed_AllFishCombined_unfiltered.csv).  
-3. A preliminary filtering of the data to remove values that had large TS compensation applied. This analysis ([ExploratoryAnalysis/generating_filtered_compensated_targets.R](ExploratoryAnalysis/generating_filtered_compensated_targets.R)) provides a list of single target detections that have <6 dB compensation that is used to filter the dataset for analysis. The filtered file is [ProcessedData/processed_AnalysisData.csv](ProcessedData/processed_AnalysisData.csv).  
-  
+2. The exported acoustic data is compiled to a master data set using [Analysis_Scripts/read_tidy_export_EVfiles.R](Analysis_Scripts/read_tidy_export_EVfiles.R). This script will import and format each fish's exported data and then compile a large dataframe *ProcessedData/processed_AllFishCombined_unfiltered.csv*.  
+3. A preliminary filtering of the data to remove values that had large TS compensation applied. This analysis [ExploratoryAnalysis/generating_filtered_compensated_targets.R](ExploratoryAnalysis/generating_filtered_compensated_targets.R)) provides a list of single target detections that have <6 dB compensation that is used to filter the dataset for analysis. The filtered file is *ProcessedData/processed_AnalysisData.csv*.  
+
+``` r
```

Comment:
what's the purpose of having this source code here?

Reply:
running those two lines of code will/should? build the raw datasets now that they are no longer part of the repo as an LFS file. Each user has to clone the repo and build the large files locally.

Reply:
I see, ya. I think it just needs some text as an explainer then - to create the above files you just need to run this code chunk in your R console

Reply:
how about with these additions?

Reply:
yup, looks good to me